core/DAPS/requests/README.md

```diff
@@ -0,0 +1,9 @@
+# DAPS requests
+
+|**request**|**example**|**summary**|
+|:---|:---|:---|
+|**[POST root](./root_POST.md)** |                                       | get Dynamic Attribute Token (DAT) |
+|                                | [1 request](./root_POST.md#1-request) | get Dynamic Attribute Token (DAT) |
+||||
+
+---
```

Comment:
This is not finished, is it?

Reply:
This is ONLY a suggestion from my side to get an idea, how to show an request (in a generic manner...). You can see it under Broker-stuff, also...